app/models/session_date.rb

```diff
@@ -23,6 +23,7 @@ class SessionDate < ApplicationRecord
   belongs_to :session
 
   has_many :session_attendances, dependent: :restrict_with_error
+  has_many :pre_screenings
```

Comment:
I think we should add `restrict_with_error` to this as well, if a session date is deleted and there exists pre-screenings should we prevent the deletion of the session date.